### PR TITLE
New airlock sounds.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
For whatever reason, the PR author randomly closed the PR and deleted their entire repo. Luckily I had it downloaded. These have been testmerged for the last 2-3 weeks.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -200,34 +199,39 @@
...
+	open_sound_powered = 'sound/machines/airlock/hall3o.ogg'
+	close_sound_powered = 'sound/machines/airlock/hall3c.ogg'
 
 /obj/machinery/door/airlock/vaurca
 	name = "Alien Biomass Airlock"
 	icon = 'icons/obj/doors/Doorvaurca.dmi'
 	opacity = TRUE
 	hatch_colour = "#606061"
 	hashatch = FALSE
+	open_sound_powered = 'sound/machines/airlock/scp1o.ogg'
```
Comment (SleepyGemmy) [inline code/game/machinery/doors/airlock.dm:224]: sound doesn't exist anymore.

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -200,34 +199,39 @@
...
+	close_sound_powered = 'sound/machines/airlock/scp1c.ogg'
```
Comment (SleepyGemmy) [inline code/game/machinery/doors/airlock.dm:225]: ditto.

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -505,6 +534,8 @@ obj/machinery/door/airlock/glass_centcom/attackby(obj/item/I, mob/user)
 	maxhealth = 600
 	insecure = 0
 	hashatch = FALSE
+	open_sound_powered = 'sound/machines/airlock/scp1o.ogg'
```
Comment (SleepyGemmy) [inline code/game/machinery/doors/airlock.dm:537]: ditto.

File: code/game/machinery/doors/airlock.dm [hunk 4]
```diff
@@ -505,6 +534,8 @@ obj/machinery/door/airlock/glass_centcom/attackby(obj/item/I, mob/user)
...
+	close_sound_powered = 'sound/machines/airlock/scp1c.ogg'
```
Comment (SleepyGemmy) [inline code/game/machinery/doors/airlock.dm:538]: ditto.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]
changes addressed.

**Review by mikomyazaki — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**